spec/cell_spec.rb

```diff
@@ -0,0 +1,38 @@
+require './lib/cell'
+require './lib/grid'
+
+describe Cell do
+
+  let(:cell) {Cell.new('1')}
+  let(:cell0) {Cell.new('0')}
+  let(:grid) {Grid.new('005000009009000000000000000000000000000000000000000000000000000000000000000000000')}
```

Comment:
Line is too long. [108/80]<br>Space missing inside {.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.